searx/static/themes/simple/src/less/preferences.less

```diff
@@ -72,6 +72,15 @@
     font-size: 90%;
   }
 
+  .bang {
+    text-align: left;
```

Comment:
Ahh.. I forgot: it should `.ltr-text-align-left()` so it works also with the RTL languages.

Reply:
fixed ... if you think PR is OK, press merge button .. / I have to go for now / thanks for your improvements :+1: